### DISCUSSION TITLE
Problem in Node-Red

### DISCUSSION THREAD

**Discussion by dcwetr** [thebigpotatoe/node-red-contrib-face-recognition#33] [Q&A]
Hi thebigpotatoe! we're students at HSLU in Switzerland and having some trouble implementing your flow for an use case we're currently testing. Could you help us? 

we installed everything according this page https://flows.nodered.org/node/node-red-contrib-face-recognition 
by executing, this issue shows up: 
![image](https://github.com/thebigpotatoe/node-red-contrib-face-recognition/assets/113893543/e78fa191-7fd4-425e-8d3e-4afbe4d22f8d)

by adding some own "Decriptors" and pictures node red crashes
![image](https://github.com/thebigpotatoe/node-red-contrib-face-recognition/assets/113893543/90ed5d59-43a0-471a-bf80-61d64dc17d28)
![image](https://github.com/thebigpotatoe/node-red-contrib-face-recognition/assets/113893543/efd31a96-e736-435d-b844-dcd8dbda865a)
![image](https://github.com/thebigpotatoe/node-red-contrib-face-recognition/assets/113893543/a9ea0b29-bcf5-4a18-8ff8-1165c983651e)

thank you very much in advance for your help!

**Comment by thebigpotatoe** [comment]
Hi guys,

It's been a long while since this code has been touched but should still run if you set everything back to what it was when the code was written. 

It will also depend on your runtime. Are you self hosting node red? It would help a lot of you could get the full logs of what's happening in the background for analysis too.
